src/linker/System.Diagnostics.CodeAnalysis/DynamicDependencyAttribute.cs

```diff
@@ -0,0 +1,136 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+namespace System.Diagnostics.CodeAnalysis
+{
+	/// <summary>
+	/// States a dependency that one member has on another.
+	/// </summary>
+	/// <remarks>
+	/// This can be used to inform tooling of a dependency that is otherwise not evident purely from
+	/// metadata and IL, for example a member relied on via reflection.
+	/// </remarks>
+	/// <remarks>
+	/// This is a copy of the enum definition in the framework at https://github.com/dotnet/runtime/blob/master/src/libraries/System.Private.CoreLib/src/System/Diagnostics/CodeAnalysis/DynamicDependencyAttribute.cs
+	/// </remarks>
+	[AttributeUsage (
+		AttributeTargets.Constructor | AttributeTargets.Field | AttributeTargets.Method,
+		AllowMultiple = true, Inherited = false)]
+	internal sealed class DynamicDependencyAttribute : Attribute
```

Comment:
Since you have the internal version of this - why do we need a copy of the declaration now?

Reply:
Now it's only being used for `IsAttribute<T>`. If you prefer, I can get rid of that and use `IsDynamicDependencyAttribute` instead - let me know.

Reply:
I don't mind keeping it, but I would only do it as an empty shell - to make it clear - we're going to delete it once we move over to 5 anyway.